docs/rfc/0006_multiscan.md

```diff
@@ -0,0 +1,128 @@
+|              |                                 |
+| :----------- | :------------------------------ |
+| Feature Name | Multi-Scan                      |
+| Start Date   | 6 November 2025                 |
+| Category     | Architecture                    |
+| RFC PR       | [fill this in after opening PR] |
+| State        | **ACCEPTED**                    |
+
+# Summary
+
+[summary]: #summary
+
+Support grype as an additional tool to scan for SBOMs.
+
+# Motivation
+
+[motivation]: #motivation
+
+We want to add support for `grype` in order to enrich the vulnerability reports, making them more complete and accurate.
+
+This will allow us to be vendor-neutral, since we are currently relying only on `trivy` to generate SBOMs and scan for vulnerabilities.
+
+Additionally, we discovered that `grype` is able to find more vulnerabilities than `trivy`. Below is a recap of our research:
+
+| image | `trivy` | `grype` |
+|-------|---------|---------|
+| `golang:1.12-alpine` | 45   | 210 |
+| `nginx:1.21.0` | 396 | 522 |
+| `redis:6.2.0-alpine` | 44 | 127 |
+| `postgres:13.0-alpine` | 63 | 151 |
+
+## Examples / User Stories
+
+[examples]: #examples
+
+### User story 1
+
+As a user, I want to make use of KEV and the EPSS score (which are currently provided by grype) to prioritize vulnerability remediation efforts.
+
+# Detailed design
+
+[design]: #detailed-design
+
+For this new feature, we are providing a way to enable it when scanning.
+
+This will impact the `ScanJob` CRD, adding a new boolean field called `multiScan`, set to `false` by default.
+
+To enable it, the `ScanJob` should be set like this:
+
+```yaml
+apiVersion: sbomscanner.kubewarden.io/v1alpha1
+kind: ScanJob
+metadata:
+  name: scanjob-example
+  namespace: default
+spec:
+  registry: example-registry
+  multiScan: true
```

Comment:
Note: that poses a question about which scanner is the default one when `multiScan` is disabled.

Reply:
yes, but let's clarify that inside of the RFC

Reply:
I am not sure whether this setting belongs at the Registry level or at the Installation level, or if we should even add it in the first iteration.

I also don’t think a `multiScan` field is explicit enough.
It would be clearer to let users select the engines, for example:
```
scanEngines: [trivy, grype]
```